src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -49,7 +50,19 @@ class ModuleController extends FrameworkBundleAdminController
      */
     public function catalogAction()
     {
+        if ((! $this->isGranted(PageVoter::READ, 'ADMINMODULESSF_')) &&
+            (! $this->isGranted(PageVoter::UPDATE, 'ADMINMODULESSF_')) &&
+            (! $this->isGranted(PageVoter::CREATE, 'ADMINMODULESSF_'))) {
+
```

Comment:
It is way better and readable to write:

```php
if (
    !$this->isGranted(PageVoter::READ, 'ADMINMODULESSF_')
    && !$this->isGranted(PageVoter::UPDATE, 'ADMINMODULESSF_')
    && !$this->isGranted(PageVoter::CREATE, 'ADMINMODULESSF_')
) {
```

And the brackets are useless just for the `!`.